src/main/java/org/mifos/processor/bulk/config/RegisteringInstitutionConfig.java

```diff
@@ -0,0 +1,26 @@
+package org.mifos.processor.bulk.config;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+@Component
+@ConfigurationProperties(prefix = "budget-account.registering-institutions")
+public class RegisteringInstitutionConfig {
```

Comment:
@arkadasfynarfin this is the one level nested config for 2D config.

Reply:
@danishjamal104 This is nicely done.